Tests/Armonik.Windows.Tests/Program.cs

```diff
@@ -0,0 +1,256 @@
+
+      var numbers = new List<double>
+      {
+        1.0,
+        2.0,
+        3.0,
+        3.0,
+        3.0,
+        3.0,
+        3.0,
+        3.0,
+      }.ToArray();
+
+      var handler = new ResultHandler(logger_);
+
+      sessionService.Submit("ComputeBasicArrayCube",
+                            ParamsHelper(numbers),
+                            handler);
+
+      sessionService.Submit("ComputeReduceCube",
+                            ParamsHelper(numbers),
+                            handler);
+
+      sessionService.Submit("ComputeReduceCube",
+                            ParamsHelper(numbers.SelectMany(BitConverter.GetBytes).ToArray()),
+                            handler);
+
+      sessionService.Submit("ComputeMadd",
+                            ParamsHelper(numbers.SelectMany(BitConverter.GetBytes).ToArray(),
+                                         numbers.SelectMany(BitConverter.GetBytes).ToArray(),
+                                         4.0),
+                            handler);
+
+      sessionService.Submit("NonStaticComputeMadd",
+                            ParamsHelper(numbers.SelectMany(BitConverter.GetBytes).ToArray(),
+                                         numbers.SelectMany(BitConverter.GetBytes).ToArray(),
+                                         4.0),
+                            handler);
+    }
+
+
+    private static object[] ParamsHelper(params object[] elements)
+    {
+      return elements;
+    }
+
+    /// <summary>
+    ///   Initialize Setting for task i.e :
+    ///   Duration :
+    ///   The max duration of a task
+    ///   Priority :
+    ///   Work in Progress. Setting priority of task
+    ///   AppName  :
+    ///   The name of the Application dll (Without Extension)
+    ///   VersionName :
+    ///   The version of the package to unzip and execute
+    ///   Namespace :
+    ///   The namespace where the service can find
+    ///   the ServiceContainer object develop by the customer
+    /// </summary>
+    /// <returns></returns>
+    private static TaskOptions InitializeSimpleTaskOptions()
+    {
+      TaskOptions taskOptions = new TaskOptions
+      {
+        MaxDuration = new Duration
+        {
+          Seconds = 3600 * 24,
+        },
+        MaxRetries = 3,
+        Priority = 1,
+      };
+
+      taskOptions.Options.Add(AppsOptions.EngineTypeNameKey,
+                              EngineType.DataSynapse.ToString());
+
+      taskOptions.Options.Add(AppsOptions.GridAppNameKey,
+                              "ArmoniK.Samples.GridServer.Services");
+
+      taskOptions.Options.Add(AppsOptions.GridAppVersionKey,
+                              "1.0.0-700");
+
+      taskOptions.Options.Add(AppsOptions.GridAppNamespaceKey,
+                              "ArmoniK.Samples.GridServer.Client.Services");
+
+      taskOptions.Options.Add(AppsOptions.GridServiceNameKey,
+                              "ServiceContainer");
```

Comment:
Use the proper fields in the taskoptions

Reply:
ok